cogeo_mosaic/backends/file.py

```diff
@@ -53,9 +53,17 @@ def write(self, gzip: bool = None):
             else:
                 f.write(json.dumps(body).encode("utf-8"))
 
-    def update(self):
+    def update(
+        self,
+        features: Sequence[Dict],
+        accessor: Callable,
+        overwrite: bool = False,
+        **kwargs: Any
+    ):
         """Update the mosaicjson document."""
-        raise NotImplementedError
+        self._update(features, accessor, **kwargs)
+        if overwrite:
+            self.write()
```

Comment:
`backend.update` handles mosaicjson update and overwriting.

Reply:
So if `overwrite` is `False`, that means update the internal mosaicjson document but don't write it to the backend?

Reply:
yes